src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Controllers/AuthenticationController.cs

```diff
@@ -153,13 +161,12 @@ public async Task<ActionResult> RefreshJwtCookie()
         }
 
         /// <summary>
-        /// Action for converting a JWT generated by <c>Maskinporten</c> with a new JWT for further use as authentication against rest of Altinn.
+        /// Action for exchanging a JWT generated by a trusted token provider with a new JWT for further use as authentication against rest of Altinn.
         /// </summary>
-        /// <returns>The result of the action. Contains the new token if the old token was valid and could be converted.</returns>
+        /// <returns>The result of the action. Contains the new token if the old token was valid and could be exchanged.</returns>
         [AllowAnonymous]
-        [HttpGet("convert")]
-        public async Task<IActionResult> AuthenticateOrganisation(
-            [FromQuery] bool test)
+        [HttpGet("exchange/{tokenProvider}")]
```

Comment:
Changing the URL of the existing endpoint is going to cause issues for both Designer and Altinn CLI. Maybe more? This is a breaking change and I'm not sure it is worth the hassle.

Reply:
I checked the CLI-application.. It seems that goes directly towards maskinporten and not the authentication component.. Could you point me to where this endpoint is used?

Reply:
@SandGrainOne The change in designer is only updating the appsettings and values file, I think that's a quick fix. I'll add it to this PR

Reply:
The CLI application use the convert endpoint in the AuthorizationClientWrapper class.

Reply:
Updated the CLI code and resolving conversation